src/test/java/com/google/cloud/tools/appengine/operations/DevServerJava8Test.java

```diff
@@ -148,6 +147,7 @@ public void testPrepareCommand_allFlags() throws Exception {
                 .projectId("my-project")
                 .environment(ImmutableMap.of("ENV_NAME", "ENV_VAL"))
                 .additionalArguments(Arrays.asList("--ARG1", "--ARG2"))
+                .projectJdkVersion("1.8")
```

Comment:
Is this change required for the existing tests to pass?

Reply:
This was added because of `SpyVerifier.newVerifier(configuration).verifyAllValuesNotNull();` checking that all the flags are non-null. Added this so the `_allFlags()` test would pass.